src/lib/TranscriptEditor/MediaPlayer/PlayerControls.js

```diff
@@ -76,7 +76,11 @@ PlayerControls.propTypes = {
   timecodeOffset: PropTypes.string,
   promptSetCurrentTime: PropTypes.func,
   rollback: PropTypes.func,
-  handleMuteVolume: PropTypes.handleMuteVolume
+  handleMuteVolume: PropTypes.handleMuteVolume,
```

Comment:
Should probably be a function?

```suggestion
   handleMuteVolume: PropTypes.func,
```

Otherwise raises error 
`Warning: Failed prop type: PlayerControls: prop type `handleMuteVolume` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.`